storage/cql_iface/cql_iface.py

```diff
@@ -61,9 +37,16 @@ def check_values_from_definition(definition):
                 CQLIface.check_values_from_definition(v)
         else:
             try:
-                CQLIface._hecuba2cassandra_typemap[definition]
-            except KeyError:
-                raise TypeError(f"The type {definition} is not supported")
+                if isinstance(definition.__origin__, (Tuple, FrozenSet)):
```

Comment:
`check_values_from_definition`should be a private method, (start with `_`). Because you don't want users thinking this is part of the API